### PR TITLE
Addes checker code for DEREF_AFTER_NULL

### PR DESCRIPTION
**Pull request by sgchoi5**
Needs to resolve Svace issues for C# Samples
1) Added null checker code to fix DEREF issues

Report for DEREF
https://code.sec.samsung.net/ahub/service/analyses/190676
https://code.sec.samsung.net/ahub/service/analyses/191290

Result for patch code
https://code.sec.samsung.net/ahub/service/analyses/191795
- Critical issues related with DEREF are removed.

### CODE REVIEW

File: TV/MediaHubSample/MediaHubSample/src/View/ImageItem.cs [hunk 1]
```diff
@@ -285,14 +285,16 @@ protected override void Dispose(DisposeTypes type)
                 //Dispose the main text
                 if (mainText != null)
                 {
-                    rootView.Remove(mainText);
+                    if (rootView != null)
+                        rootView.Remove(mainText);
```
Comment (jkpu) [inline TV/MediaHubSample/MediaHubSample/src/View/ImageItem.cs:289]: How about to change to

 rootView?.Remove(mainText);

File: TV/MediaHubSample/MediaHubSample/src/View/ImageItem.cs [hunk 2]
```diff
@@ -285,14 +285,16 @@ protected override void Dispose(DisposeTypes type)
...
                     mainText.Dispose();
                     mainText = null;
                 }
                 //Dispose the main image
                 if (mainImage != null)
                 {
-                    rootView.Remove(mainImage);
+                    if (rootView != null)
```
Comment (jkpu) [inline TV/MediaHubSample/MediaHubSample/src/View/ImageItem.cs:296]: same above

rootView?.Remove(mainImage);

File: TV/MediaHubSample/MediaHubSample/src/View/ImageItem.cs [hunk 3]
```diff
@@ -301,7 +303,8 @@ protected override void Dispose(DisposeTypes type)
                 {
                     for (int i = 0; i < informationIconArray.Length; i++)
                     {
-                        rootView.Remove(informationIconArray[i]);
```
Comment (jkpu) [inline TV/MediaHubSample/MediaHubSample/src/View/ImageItem.cs:304]: rootView?.Remove(informationIconArray[i]);

File: Wearable/Ultraviolet/src/Ultraviolet/Ultraviolet/Ultraviolet.Tizen.Wearable/Services/UltravioletSensorService.cs [hunk 4]
```diff
@@ -53,7 +53,7 @@ public UltravioletSensorService()
         public void Start()
         {
             _ultravioletSensor.DataUpdated += OnDataUpdated;
-            _ultravioletSensor?.Start();
```
Comment (jkpu) [inline Wearable/Ultraviolet/src/Ultraviolet/Ultraviolet/Ultraviolet.Tizen.Wearable/Services/UltravioletSensorService.cs:56]: following code is more proper.

        public void Start()
        {
            if(_ultravioletSensor != null)
            {
                _ultravioletSensor.DataUpdated += OnDataUpdated;
                _ultravioletSensor.Start();
            }
        }

Comment (sgchoi5) [reply Wearable/Ultraviolet/src/Ultraviolet/Ultraviolet/Ultraviolet.Tizen.Wearable/Services/UltravioletSensorService.cs:56]: For code consistency, Start() method was fixed like Stop().
 - Start() used ? to check null object partially. 
 - Stop() did not use ? to check null object.

File: Mobile/UI/Puzzle/Puzzle/Puzzle.Tizen.Mobile/Renderers/CustomImageButtonRenderer.cs [hunk 5]
```diff
@@ -52,7 +52,8 @@ protected override void OnElementChanged(ElementChangedEventArgs<CustomImageButt
 			if (Control == null)
 			{
 				SetNativeControl(new Box(Forms.NativeParent));
-				Control.SetLayoutCallback(OnLayout);
+				if (Control != null)
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply Mobile/UI/Puzzle/Puzzle/Puzzle.Tizen.Mobile/Renderers/CustomImageButtonRenderer.cs:55]: OK. Thanks.

File: TV/MediaHubSample/MediaHubSample/src/View/ImageItem.cs (hunk 1 above)
Comment (sgchoi5) [reply TV/MediaHubSample/MediaHubSample/src/View/ImageItem.cs]: Thanks. Fixed.

File: TV/MediaHubSample/MediaHubSample/src/View/ImageItem.cs (hunk 2 above)
Comment (sgchoi5) [reply TV/MediaHubSample/MediaHubSample/src/View/ImageItem.cs]: Thanks. Fixed.

File: TV/MediaHubSample/MediaHubSample/src/View/ImageItem.cs (hunk 3 above)
Comment (sgchoi5) [reply TV/MediaHubSample/MediaHubSample/src/View/ImageItem.cs:304]: Thanks. Fixed.

**Review by jkpu — APPROVED** [review]

**Review by Juwols — APPROVED** [review]
Looks good to me

**Pull request merged**